Signal/src/view controllers/CallViewController.swift

```diff
@@ -662,8 +662,6 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
     func didPressVideo(sender: UIButton) {
         Logger.info("\(TAG) called \(#function)")
         let hasLocalVideo = !sender.isSelected
-        audioModeVideoButton.isSelected = hasLocalVideo
```

Comment:
Why remove this? I guess because it's redundant with updateCallUI? 
We're not handling the mute/speaker buttons in that way though.

Not a blocking concern at all, just wondering if you had a design goal here.